tracee-rules/signatures/signaturestest/signaturestest.go

```diff
@@ -1,32 +1,35 @@
 package signaturestest
 
-import "github.com/aquasecurity/tracee/tracee-rules/types"
+import (
+	"github.com/aquasecurity/tracee/tracee-rules/types"
+)
 
-// SigTest is a utility to test signatures
-type SigTest struct {
-	// Events are a series of events that the signature will be tested with (in the provided order)
-	Events []types.Event
-	// CB provide a callback for this test case only if you want to check the Finding result
-	// if providing your own callback, make sure to call SetStatus from it
-	CB types.SignatureHandler
```

Comment:
Tests based on callbacks are kind of fragile. It's easy to forget to invoke the callback which might give false positive / negative test. It's simpler to assert to data (= Go structures) not functions.